src/youtube-player/youtube-player.spec.ts

```diff
@@ -37,7 +37,7 @@ describe('YoutubePlayer', () => {
     });
 
     afterEach(() => {
-      delete window.YT;
+      (window as any).YT = undefined;
```

Comment:
Is this necessary because in TS v4, operands for `delete` need to be optional and the Youtube types declare `YT` _always_  on `window`?

Reply:
That is correct